src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
+});
+
+export default categoriesSlice.reducer;
```

Comment:
Well done @preciousbetine 

You did everything well except that I do not see how you exported the actions using the ES6 object destructuring from ``categoriesSlice.actions``. 

```javascript
  export const { checkStatus } = categoriesSlice.actions
```

Kindly add it

Reply:
Hi @bobb-Rob , thanks for taking the time to review my work!  Just to clarify, exporting the slice's actions wasn't explicitly stated as a requirement for this project. Nonetheless, I will add it and re-request a review.

Reply:
@bobb-Rob Thanks. Must have been an oversight